pkgs/tools/package-management/reuse/default.nix

```diff
@@ -21,6 +21,7 @@ buildPythonApplication rec {
     license-expression
     requests
     setuptools
+    setuptools_scm
```

Comment:
Perhaps all the Python libraries should be included on their own and not the whole `python3Packages` in the first line. This was already the case before, but could perhaps be changed.

Reply:
This PR is just the version bump; I would prefer to keep it single-purpose.

Reply:
Okay, but perhaps someone should have another look at this later.